### DISCUSSION TITLE
Incoherent Pydantic definition in documentation

### DISCUSSION THREAD

**Discussion by kauedg** [fastapi/fastapi#6559] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
No code, it's a documentation correction (possibly).
```

### Description

FastAPI's documentation, in it's "[Python Types/Pydantic models](https://fastapi.tiangolo.com/python-types/#pydantic-models)" section says:

> Pydantic is a Python library to perform data validation.
>
>You declare the "shape" of the data as classes with attributes.
>
>And each attribute has a type.
>
>Then you create an instance of that class with some values and it will validate the values, convert them to the appropriate type (if that's the case) and give you an object with all the data.

But Pydantic's documentation, in the "[Models](https://pydantic-docs.helpmanual.io/usage/models/)" section notes that:
> pydantic is primarily a parsing library, **not a validation library**. Validation is a means to an end: building a model which conforms to the types and constraints provided.

Then FastAPI's documentation could be rewritten this way:

> Pydantic is a Python library to perform data parsing.
>
>You declare the "shape" of the data as classes with attributes.
>
>And each attribute has a type.
>
>Then you create an instance of that class with some values and it will parse the values, convert them to the appropriate type (if that's the case) and give you an object with all the data.

I did not send a pull request right away because I think it may have been written this way to facilitate the understanding but I think that it's better to be technically correct.

### Operating System

Linux

### FastAPI Version

0.68.0

### Python Version

irrelevant

**Comment by Kludex** [comment]
Duplicated https://github.com/tiangolo/fastapi/issues/3095
